user-management-and-permissions_5c07ccc604286304a71d0252.md

```diff
@@ -12,13 +12,15 @@ Owner can:
 
 - See all projects;
 - Add projects, if they have admin access to the repository on GitHub;
-- Delete projects.
+- Delete projects if they have admin access to the repository on Github;
+- Manage people;
+- Manage billing;
 
-### Owner's area
+### Owner Owner's area access
```

Comment:
Are we seriously calling it Owner Owner? :)

Reply:
A little bit unfortunate ;)